### ISSUE TITLE
flowchart.js integration with angular

### ISSUE THREAD

**Issue by ChiradeepNanabala** [adrai/flowchart.js#199]
I am trying to integrate flowchart.js with angular. I have included all the required files. But, I am getting some error like this..

zone-evergreen.js:172 Uncaught TypeError: Raphael is not a constructor
    at new FlowChart (flowchart-latest.js:955)
    at Object.drawSVG (flowchart-latest.js:675)
    at btn.onclick (check.js:14)
    at window.onload (check.js:57)
    at wrapFn (zone-evergreen.js:1202)
    at ZoneDelegate.invokeTask (zone-evergreen.js:400)
    at Zone.runTask (zone-evergreen.js:168)
    at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:481)
    at invokeTask (zone-evergreen.js:1596)
    at globalZoneAwareCallback (zone-evergreen.js:1633)
Can anyone find the issue..
Thanks in Advance.

**Comment by adrai** [comment]
Have you included the raphael script? https://github.com/adrai/flowchart.js/blob/master/example/index.html#L9

**Comment by ChiradeepNanabala** [comment]
Yes, I have included it. Please find the attachment of my angular app

[app files.zip](https://github.com/adrai/flowchart.js/files/4202872/app.files.zip)

**Comment by adrai** [comment]
Sorry, I'm not familiar with angular...
but when I open your files, I get:
![image](https://user-images.githubusercontent.com/1086194/74527437-f91c7f00-4f25-11ea-81b0-19de53d5b936.png)

**Comment by ChiradeepNanabala** [comment]
How did you execute the above files? In angular or just opened normal html file?
I am not getting the output in angular...!

**Comment by adrai** [comment]
I opened the served html

**Comment by adrai** [comment]
Can you create a codesandbox example?
